engine/include/engine/graphics/GraphicsController.hpp

```diff
@@ -151,6 +151,21 @@ namespace engine::graphics {
             return m_ortho_params;
         }
 
+
+        unsigned int get_vbo();
+        unsigned int get_vao();
+        unsigned int get_rbo();
+        unsigned int get_framebuffer();
+        unsigned int get_texture_colorbuffer();
+
+        void set_vao(unsigned int value);
+        void set_rbo(unsigned int value);
+        void set_vbo(unsigned int value);
+        void set_framebuffer(unsigned int value);
+        void  set_texture_colorbuffer(unsigned int value);
+
+        // same for setters
+
```

Comment:
Nisam siguran da u potpunosti razumem razlog zašto se vbo, vao, rbo, framebuffer i texture_colorbuffer čuvaju unutar GraphicsController klase.

Čini mi se da ovo možda nije dovoljno generalno da bi bilo deo engine modula, s obzirom na to da GraphicsController trenutno čuva samo jedan VertexBufferObject, VertexArrayObject, RenderBufferObject, Framebuffer i ColorBuffer. Ideja Engine modula je da bude opšti i ponovo upotrebljiv, tako da ga sve aplikacije mogu koristiti bez potrebe za izmenama. Možda bi bilo korisno razmotriti alternativnu organizaciju?